data/human/boarding missions.txt

```diff
@@ -27,7 +27,8 @@ mission "Assisting Merchant (Small)"
 		not attributes "automaton"
 	on offer
 		payment 10000
-		conversation "assisting merchant"
+		conversation
```

Comment:
This "assisting merchant" conversation you're replacing is defined above this mission. Is it not used anywhere now that you've removed it from these three missions? Is there any reason you've couldn't change the conversation to contain the new text instead of copy/pasting the text to each of these missions?